pkgs/development/python-modules/pywal16/default.nix

```
@@ -0,0 +1,56 @@
+# This package is almost copy-paset of original Pywal's
+# So, I give credit to Fresheyeball, as to maintainer
+# Please, feel free to maintain this (I don't know what I'm doing)
+
+
+{ lib,
+  python3,
+  fetchPypi,
+  imagemagick,
+  swww, #feh
+  ...
+}:
+
+python3.pkgs.buildPythonPackage rec {
+  pname = "pywal16";
+  version = "3.5.3";
+  format = "setuptools";
```

```suggestion
{ lib,
  buildPythonPackage,
  fetchPypi,
  imagemagick,
  swww, #feh
  ...
}:

buildPythonPackage rec {
  pname = "pywal16";
  version = "3.5.3";
  format = "setuptools";
```

Packages in `python-modules` should get called as `pkgs.python3.pkgs.callPackage`, and thus should take anything from the `python3.pkgs` scope as arguments directly. You can look at the other pywal package for an example: https://github.com/NixOS/nixpkgs/blob/b7d845292c304e026d86097e6d07409070e80dcc/pkgs/development/python-modules/pywal/default.nix